docs/process/product.md

```diff
@@ -0,0 +1,145 @@
+# 💾 Product
+
+This document exists as a comprehensive version of Flexpa's product development process, including meetings, artifacts, and workflows.
+
+<figure><img src="../../.gitbook/assets/product-flow.jpg" alt=""><figcaption></figcaption></figure>
+
+## OKRs
+
+**Purpose**: Objectives or underlying KRs are the primary way we set goals at Flexpa - including product development goals.
+
+**Output**:
+
+- Quarterly collaboration with other roles on a list of objectives and associated key results that serve as measurable goals for the team
+- Product OKRs defined in Github with an OKR tag
```

Comment:
Are we still doing this? I was going to point to the Linear Project board in the updated Roadmap doc https://linear.app/flexpa/team/PRD/projects/all

Reply:
Yeah that's a good catch, we definitely are not using that tag

For the product dev page it could link to the product team projects (I'll update)

But the root page for the overall OKRs is: https://linear.app/flexpa/initiatives